eslintrc

```diff
@@ -80,7 +79,7 @@ rules:
   no-sequences: 2
   no-shadow: 2
   no-shadow-restricted-names: 2
-  no-space-before-semi: 2
+  semi-spacing: [2, {before: false, after: false}
```

Comment:
I think we want `after: true` or for loops will all break. That is the default, so you can just delete the options.